packages/fluxible/tests/unit/lib/FluxibleContext.js

```diff
@@ -522,9 +540,17 @@ describe('FluxibleContext', function () {
                         context: context,
                         payload: payload
                     });
-                    callback();
+                    context.executeAction(actionFour, payload, callback);
                 };
                 actionThree.displayName = 'Three';
+                var actionFour = function (context, payload, callback) {
+                    actionCalls.push({
+                        context: context,
+                        payload: payload
+                    });
+                    callback();
+                };
+                actionFour.displayName = 'Four';
```

Comment:
@mridgway can you also try with action four calling another deeper action?

`One.Four.Six` case